padl/transforms.py

```diff
@@ -1641,17 +1759,16 @@ def _pd_classify_nodetype(i, t, t_m1, cw, cw_m1):
 
         return type_
 
-    def _pd_longrepr(self, formatting=True) -> str:  # TODO: make it respect the formatting
+    def _pd_longrepr(self, formatting=True, marker=None) -> str:  # TODO: make it respect the formatting
         """Create a detailed formatted representation of the transform. For multi-line inputs
         the lines are connected with arrows indicating data flow.
         """
         # pad the components of rows which are shorter than other parts in same column
         rows = [
             [s._pd_parensrepr() for s in t.transforms] if hasattr(t, 'transforms')
             else [t._pd_shortrepr()]
-            for t in self.transforms
+            for i, t in enumerate(self.transforms)
```

Comment:
I don't think the `enumerate` is needed here. Could you change it back?

Reply:
True